setup.py

```diff
@@ -1,67 +1,15 @@
-import importlib.util
-import os.path
-from setuptools import setup, find_packages
+import pathlib
+import re
+import setuptools
 
 
-HERE = os.path.abspath(os.path.dirname(__file__))
+if __name__ == '__main__':
+    # NOTE(vytas): It is unfortunate to resort to the below regex, but
+    #   apparently attribute lookup via setup.cfg is very fragile otherwise.
+    here = pathlib.Path(__file__).parent
+    with open(here / 'falcon_sqla' / 'version.py') as version_py:
```

Comment:
I've used this in multiple places, and iirc it's >=47, that's a 2 year old version, so it should not be too bad.

In any case the regexp version is also always fine

Reply:
I need to move `__version__` to `__init__.py`, and even then it was failing on CPython 3.6 pipeline because it probably had older setuptools by default. I'll circle back on this later, cause for now I had to move the package's name back to `setup.py` as well in order to show dependents on GitHub... (Which can be fixed via GH support... but I'd rather spend energy adding async session.)